recipes/qt/5.x.x/conanfile.py

```diff
@@ -507,9 +507,7 @@ def build(self):
             args += ["-opengl dynamic"]
 
         if self.options.get_safe("with_vulkan", False):
-            args.append("-vulkan")
```

Comment:
I'm surprised that the wrong argument was used. Did it produce an error message?

Reply:
No.
But could you please try on your own, please?

I would not merge this PR until it works for MacOs.

Reply:
Then you should probably change this PR to draft. I will try to test this on Friday, or it will be next week

Reply:
@AndreyMlashkin please revert this change. the existing argument is correct. `-feature-vulkan` works too, but changing this line it makes git blame harder to use.